subprojects/docs/src/docs/userguide/declaringDependencies.adoc

```diff
@@ -15,7 +15,7 @@
 [[declaring_dependencies]]
 == Declaring Dependencies
 
-Gradle builds can declare dependencies on external binaries, raw files and other Gradle projects. You can find examples for common scenarios in this section. For more information, see the <<dependency_types,full reference on all types of dependencies>>.
+Gradle builds can declare dependencies on module, files and other Gradle projects. You can find examples for common scenarios in this section. For more information, see the <<dependency_types,full reference on all types of dependencies>>.
```

Comment:
:x: should be _modules_ with a **s**. Also, as this is the introduction, we could be a bit more descriptive. Something like: 

_modules hosted in repositories_

Reply:
Thanks, done